### PR TITLE
Fix CI: Add lint-python endpoint

### PR DESCRIPTION
**Pull request by nottrobin**
See the CI pass

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
     "serve": "./entrypoint 0.0.0.0:$PORT",
     "build": "echo 'nothing to build'",
-    "test": "flake8 webapp && black --line-length 79 webapp"
+    "lint-python": "flake8 webapp && black --line-length 79 webapp",
```
Comment (tbille) [inline package.json]: `black --check --line-length 79 webapp`

Comment (hatched) [inline package.json]: I think black needs `--check`

**Comment by tbille** [comment]
euh what

**Comment by barrymcgee** [comment]
😳

**Pull request merged**